src/utils/os.ts

```diff
@@ -2,6 +2,8 @@ import * as os from 'os';
 
 export const getOSInfo = () => {
   // If available, capture information about the OS
+
+  // @ts-ignore
```

Comment:
I'm actually deleting this file in a different PR, but TS 4.1 was upset that I was checking all these things in the if-statement that its knowledge of types told it should always be defined. Regardless of what the type says, i think it's prudent to be cautious regarding things you expect from the environment.